content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -2255,7 +2255,8 @@ Starting in v1.16, this annotation was removed in favor of
 - [`pod-security.kubernetes.io/audit-violations`](/docs/reference/labels-annotations-taints/audit-annotations/#pod-security-kubernetes-io-audit-violations)
 - [`pod-security.kubernetes.io/enforce-policy`](/docs/reference/labels-annotations-taints/audit-annotations/#pod-security-kubernetes-io-enforce-policy)
 - [`pod-security.kubernetes.io/exempt`](/docs/reference/labels-annotations-taints/audit-annotations/#pod-security-kubernetes-io-exempt)
-
+- [`validation.policy.admission.k8s.io/validation_failure`](docs/reference/labels-annotations-taints/audit-annotations/#validation-policy-admission-k8s-io-validation-failure)
```

Comment:
Thank you for your contribution! However, the link to the newly added annotation isn't quite right, as it's leading to a 404 error ([see the build preview](https://deploy-preview-45190--kubernetes-io-main-staging.netlify.app/docs/reference/labels-annotations-taints/#annotations-used-for-audit)). Below I have provided suggestion for fixing it. Hope you find it helpful!

```suggestion
- [`validation.policy.admission.k8s.io/validation_failure`](/docs/reference/labels-annotations-taints/audit-annotations/#validation-policy-admission-k8s-io-validation-failure)
```